src/templates/static.tsx

```diff
@@ -192,26 +270,55 @@ const StaticPage = ({
   const slug = mdx.fields?.slug || ""
 
   return (
-    <Container>
+    <Box w="full">
       {showPostMergeBanner && (
         <PostMergeBanner
           translationString={postMergeBannerTranslationString!}
         />
       )}
-      <Page dir={isRightToLeft ? "rtl" : "ltr"}>
+      <Flex
+        justifyContent="space-between"
+        w="full"
+        mx="auto"
+        mb={16}
+        p={8}
+        pt={{ lg: 16 }}
+        dir={isRightToLeft ? "rtl" : "ltr"}
+      >
         <PageMetadata
           title={mdx.frontmatter.title}
           description={mdx.frontmatter.description}
         />
-        <ContentContainer>
+        <Box
+          as="article"
+          maxW={theme.breakpoints.md}
```

Comment:
This can be improved for Chakra

There is a collection in the size theming called "container" that holds token values for container like objects. There is a matching value here.
```suggestion
          maxW="container.md"
```